services/src/api/model/operators.rs

```diff
@@ -91,10 +277,1148 @@ pub struct PlotResultDescriptor {
     pub bbox: Option<BoundingBox2D>,
 }
 
+impl From<geoengine_operators::engine::PlotResultDescriptor> for PlotResultDescriptor {
+    fn from(value: geoengine_operators::engine::PlotResultDescriptor) -> Self {
+        Self {
+            spatial_reference: value.spatial_reference.into(),
+            time: value.time.map(Into::into),
+            bbox: value.bbox.map(Into::into),
+        }
+    }
+}
+
+impl ResultDescriptor for PlotResultDescriptor {
```

Comment:
do we actually need/use the methods defined here?

Reply:
At some point during development of this PR these methods were called. I looked into it and this is no longer the case. [f79ac16](https://github.com/geo-engine/geoengine/pull/662/commits/f79ac16506c23a31c64317304f89efd420ce7900)